app/Http/Resources/ReportedUserResource.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+use App\Models\ReportedUser;
+use App\Http\Resources\ReportedUserReportResource;
+
+class ReportedUserResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function toArray($request)
+    {
+        return [
+            'id' => $this->id,
+            'username' => $this->username,
+            'report_amount' => ReportedUser::where('reported_user_id', $this->id)->get()->count(),
```

Comment:
Don't need the `get()` before the `count()`

Reply:
fixed,
also changed `'last_report_date' => $this->getLatestReport()->created_at->toDateTimeString(),`
to `'last_report_date' => $this->getLatestReport()->value('created_at')->toDateTimeString(),`